modules/drivers/mongo/test/metabase/driver/mongo/query_processor_test.clj

```diff
@@ -86,33 +89,43 @@
                     :mbql?       true}
                    (qp/compile
                     (mt/mbql-query attempts
-                      {:aggregation [[:count]]
-                       :filter      [:time-interval $datetime :last :month]}))))
+                                   {:aggregation [[:count]]
+                                    :filter      [:time-interval $datetime :last :month]}))))
```

Comment:
According to the `:style/indent` rule for `mbql-query`, the old indentation was correct. `mbql-query` is a macro but it doesn't really have a body like `do` or `let` or `defn`. So I'm not 100% sure the current `:style/indent` rule makes sense. I'm open to talking about changing the `:style/indent` rule, but if the old one is in place we should follow it until we change it

Reply:
This was completely unintentional. Sometimes paredit destroys the indentation.